gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -233,114 +454,59 @@ def run_translation_averaging(
                 or ill-constrained system).
             A GtsfmMetricsGroup of 1DSfM metrics.
         """
-        logger.info("Running translation averaging on %d unit translations", len(i2Ui1_dict))
-        noise_model = gtsam.noiseModel.Isotropic.Sigma(NOISE_MODEL_DIMENSION, NOISE_MODEL_SIGMA)
-        if self._robust_measurement_noise:
-            huber_loss = gtsam.noiseModel.mEstimator.Huber.Create(HUBER_LOSS_K)
-            noise_model = gtsam.noiseModel.Robust.Create(huber_loss, noise_model)
+        logger.info("Running translation averaging on {} unit translations".format(len(i2Ui1_dict)))
```

Comment:
nit: prefer logger syntax with %d, to prevent unnecessary computation if logger level is set higher